libr/bin/p/bin_dex.c

```diff
@@ -762,6 +801,29 @@ static void parse_class(RBinFile *binfile, RBinDexObj *bin, RBinDexClass *c, int
 				try_item[tries_size] tries
 				encoded_catch_handler_list handlers
 #endif
+
+
+				// TODO: parse debug info
+				// TODO: check for bof
+				ut16 regsz = r_read_le16(binfile->buf->buf+MC);
```

Comment:
also those commands may return NULL if its out of band. so i would just do an r_buf_read of 16 bytes at MC offset and pass this buffer to r_read_le* functions below